source/user-manual/manager/manual-integration.rst

```diff
@@ -43,6 +45,20 @@ b. For SysV Init:
 
   # service wazuh-manager restart
 
+
+Optional filters
+^^^^^^^^^^^^^^^^
+
+These fields are used by ``Integrator`` to apply certain filters to the alerts being processed and sent to the external platforms. 
+The following considerations must be taken into account when the filters are set:
+   
+   - The behavior of the ``<group>`` field when multiple values are defined is that of an OR operation. If any of the defined groups matches the alert group, the alert is processed.
+   - The behavior of the ``<rule_id>`` field when multiple values are defined is that of an OR operation. If any of the defined rule ids matches the alert id, the alert is processed. If the alert does not contain an id or does not match with the defined ones, then it is skipped.
+   - The behavior between the ``<rule_id>`` and ``<group>`` fields when both are defined. If the alert being processed does not match any of the set values or just the values present in one of the fields but not any of the other, it is skipped. If the alert matches at least one of the values of each filter, the alert is processed by the integration.
```

Comment:
Consider updating these having in mind the previous suggestion, if required.

Reply:
Done